applications/asset_tracker/README.rst

```diff
@@ -28,22 +29,26 @@ This data includes the GPS position, accelerometer data (the device's physical o
      - HUMID
    * - Air pressure
      - AIR_PRESS
+   * - Light sensor
+     - LIGHT
+
+On the nRF9160 DK, the application uses simulated sensor data by default, but it can be configured with Kconfig options to use real sensors to collect data. On the Thingy:91, onboard sensors are used by default. GPS is enabled by default on both the boards.
 
-By default, the application uses simulated sensor data, but it can be configured with Kconfig options to use real sensors to collect data.
+In addition to the sensor data, the application retrieves information from the LTE modem, such as the signal strength, battery voltage, and current operator.
+This information is available in nRF Cloud under the section **Cellular Link Monitor**.
 
-In addition to the sensor data, the application retrieves information about the LTE modem, such as the signal strength, battery voltage, and current operator.
-This information is available in nRF Cloud under the sensor type "DEVICE".
+The LTE Link Monitor PC tool can be used to send AT commands and receive the response.
 
-The LTE link control driver is used to establish the LTE link automatically.
-When the device is connected to the nRF Cloud, you can use the LTE Link Monitor to send AT commands and receive the response.
+By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
```

Comment:
I agree this is too thin. Since all the logic relating to updates is implemented in and dictated by the FOTA library (it is only enabled in the asset tracker), I think that is a better place for most of the technical details. I agree that how to update the firmware through nRF Cloud from a user perspective should be documented somewhere, and probably referenced from here.

This will have to be done after release, as the behavior is not yet quite stable in nRF Cloud yet, AFAIK.